editor/src/clj/editor/asset_browser.clj

```diff
@@ -454,6 +454,12 @@
         (when (and (delete selection) next)
           (select-resource! asset-browser next))))))
 
+(defn- create-template-file [new-file template]
+  (let [split-filename (re-matches #"(.*)\.(.+)" (.getName new-file))
+        name (get split-filename 1)
```

Comment:
foobar.collection -> ["foobar.collection" "foobar" "collection"]

Reply:
Maybe we should use something like `(FilenameUtils/getBaseName (.getName new-file))` instead? Feels more reliable...